svf/lib/Graphs/VFG.cpp

```diff
@@ -971,6 +971,10 @@ void VFG::updateCallGraph(PointerAnalysis* pta)
  */
 void VFG::connectCallerAndCallee(const CallICFGNode* callBlockNode, const SVFFunction* callee, VFGEdgeSetTy& edges)
 {
+    /// if the arg size does not match then we do not need to connect this parameter
+    /// unless the callee is a variadic function (the first parameter of variadic function is its paramter number)
+    if(SVFUtil::matchArgs(callBlockNode->getCallSite(), callee) == false)
```

Comment:
Can we make this at the assertion point? Otherwise, we may miss some of the caller-callee edges